web/pandas/pdeps/0008-inplace-methods-in-pandas.md

```diff
@@ -26,6 +26,9 @@ This PDEP proposes that:
 - The ``copy`` parameter will also be removed, except in constructors and in functions/methods that convert array-likes
   to pandas objects (e.g. the ``pandas.array`` function) and functions/methods that export pandas objects to other data
   types (e.g. ``DataFrame/Series.to_numpy`` method).
+    - Note: This deprecation is conditional of the fact that Copy-on-Write becomes the default. In the event that
+      Copy-on-Write[^1] does not become the default, this deprecation will not be executed, and all deprecation
+      warnings would be reverted.
```

Comment:
Given that I am removing explicit discussion of `copy` in https://github.com/lithomas1/pandas/pull/33, we could also remove that here altogether here in the abstract (if we go ahead with https://github.com/lithomas1/pandas/pull/33). 

We can still mention that this PDEP is dependent on PDEP-7 in the timeline section or so, but that way keeping the abstract shorter